test/model/typescript/date-converter.ts

```diff
@@ -5,10 +5,10 @@ import { JsonCustomConvert } from "../../../src/json2typescript/json-custom-conv
 export class DateConverter implements JsonCustomConvert<Date | null> {
     serialize(date: Date | null): any {
         if (date instanceof Date) {
-            let year = date.getFullYear();
-            let month = date.getMonth() + 1;
-            let day = date.getUTCDate();
-            return year + "-" + (month < 10 ? "0" + month : month) + "-" + (day < 10 ? "0" + day : day);
+            // Convert to standard ISO format, which is in UTC
+            const isoString = date.toISOString();
+            // Split on the "T" separator and return just the date portion
+            return isoString.split("T")[0];
```

Comment:
Does this work in Safari as well? I have experienced issues in Safari with the Date object when not having UTC time. That said, the original code surely has the same problem.

Reply:
Hm not sure, but since it's a test-only class, and the tests run on Chrome, I'm guessing it would be okay.

Reply:
Actually no. The class is exported and users can import it. :)